### DISCUSSION TITLE
video quality select only work on web pc browser and Android phone

### DISCUSSION THREAD

**Discussion by Densovan** [vidstack/player#1077] [Q&A] [answered]
video quality select is  only work on web pc browser and Android phone for Iphone it not show
anyone have that issue like me?
i work on vue  js 3

**Comment by mihar-22** [comment, marked as answer]
This is as expected, HLS is only natively supported on iOS and the quality can't be changed programatically.

**Reply by Densovan** [reply]
can you help suggest me any workarounds on this? or do we need web assembly library for this?

**Reply by mihar-22** [reply]
Not sure what you're asking, there is no workaround at the moment as there is no API for it.
